kubernetes/linux/main.sh

```diff
@@ -463,7 +468,7 @@ if [ ! -z "$podWithValidContainerId" ]; then
       # update runtime only if its not empty, not null and not startswith docker
```

Comment:
this comment is not valid anymore.

Reply:
> this comment is not valid anymore.

Thanks for catching this. Addressed this.